base.gradle

```diff
@@ -29,6 +29,7 @@ android {
 
         // Feature Flags
         buildConfigField "boolean", "END_OF_YEAR_ENABLED", "false"
+        buildConfigField "boolean", "END_OF_YEAR_REQUIRE_LOGIN", "true"
```

Comment:
Why do we want this flag for requiring login?

Reply:
iOS uses a remote FF to control whether you need an account login to see stories.
> There's a remote flag that dictates if an account is required or not to see the stories.

I have put a placeholder FF here, and try to move it to Firebase remote config if I can.